eZ/Bundle/EzPublishRestBundle/Features/ContentTypeGroup/list.feature

```diff
@@ -22,7 +22,18 @@ Feature: Read all Content Type Groups
             | dif3rent   |
             | id_nt.fier |
 
-    Scenario: Can't read any Content Type Group without an authorized user
+    Scenario: List Content Type Groups without an authorized user
         Given I don't have permissions
```

Comment:
To be frank I don't know, but I can look it up. REST delegates permissions management to the Repository.